templates/archives/stories/digital_humanities.jinja2

```diff
@@ -0,0 +1,43 @@
+{% extends 'archives/story.jinja2' %}
+
+{% block story_title %}
+Digital Humanities: an Enduring Legacy
+{% endblock %}
+
+{% block story_teaser %}
+A time without digital humanities is better known as a time without computers. Even at its
+    earliest, when the hours of computation were meticulously counted and rationed, the
+    investment of technology in humanism thrived.
+{% endblock %}
+
+{% block story_body %}
+    <p> Depending on how aware one is regarding the early <a href="/archives/story/debugging">perils of debugging</a>,
+    it's not difficult to understand exactly how valuable even a single hour on a computer was for
+    researchers at the time. At a time when computers were the answer to integrals humanly
+    impossible to compute, even an hour was worth over a thousand dollars in 1956 (that's around
+        $12,000 today!). The University of Chicago, wishing to acquire 60 more hours of computing
+        time, had offered $7,600 to $8,000 to MIT's Department. It wasn't that the purpose of
+        their use wasn't grand enough. It was that there simply wasn't enough computing time to
+        spare.
+    </p>
+
+    <p>In this way, only the topics that were truly important could be placed under
+        consideration for the allocation computational resources. What might be surprising is
+        that the projects socialists and anthropologists counted among those numbers. In 1957 in
```

Comment:
I think "social scientists" instead of "socialists" here :)

Reply:
(They're generally both.  :-D  )